packages/alfa-rules/src/sia-r46/rule.ts

```diff
@@ -31,10 +31,8 @@ export default Rule.Atomic.of<Page, Element>({
               not(isIgnored(device))
             )
           );
-
         for (const table of tables) {
```

Comment:
```suggestion

        for (const table of tables) {
```
Let's keep those empty lines.